internal/suites/example/compose/haproxy/haproxy.cfg

```
@@ -42,25 +42,17 @@ frontend fe_http
     http-request set-var(req.scheme) str(https) if { ssl_fc }
     http-request set-var(req.scheme) str(http) if !{ ssl_fc }
     http-request set-var(req.questionmark) str(?) if { query -m found }
-    http-request set-var(req.method) str(CONNECT) if { method CONNECT }
-    http-request set-var(req.method) str(GET) if { method GET }
-    http-request set-var(req.method) str(HEAD) if { method HEAD }
-    http-request set-var(req.method) str(OPTIONS) if { method OPTIONS }
-    http-request set-var(req.method) str(POST) if { method POST }
-    http-request set-var(req.method) str(TRACE) if { method TRACE }
-    http-request set-var(req.method) str(PUT) if { method PUT }
-    http-request set-var(req.method) str(PATCH) if { method PATCH }
-    http-request set-var(req.method) str(DELETE) if { method DELETE }
 
     http-request set-header X-Real-IP %[src]
-    http-request set-header X-Original-Method %[var(req.method)]
-    http-request set-header X-Original-URL %[var(req.scheme)]://%[req.hdr(Host)]%[path]%[var(req.questionmark)]%[query]
+    http-request set-header X-Forwarded-Method %[method]
+    http-request set-header X-Forwarded-Proto  %[var(req.scheme)]
+    http-request set-header X-Forwarded-Host   %[req.hdr(Host)]
+    http-request set-header X-Forwarded-URI    %[path]%[var(req.questionmark)]%[query]
 
     # be_auth_request is used to make HAProxy do the TLS termination since the Lua script
     # does not know how to handle it (see https://github.com/TimWolla/haproxy-auth-request/issues/12).
-    http-request lua.auth-request be_auth_request /api/authz/auth-request if protected-frontends
-
-    http-request redirect location https://login.example.com:8080/?rd=%[var(req.scheme)]://%[base]%[var(req.questionmark)]%[query]&rm=%[var(req.method)] if protected-frontends !{ var(txn.auth_response_successful) -m bool }
+	http-request lua.auth-intercept be_auth_request /api/authz/forward-auth HEAD * authorization,proxy-authorization,remote_user,remote-user,remote-groups,remote-name,remote-email - if protected-frontends
+    http-request redirect location %[var(txn.auth_response_location)] if protected-frontends !{ var(txn.auth_response_successful) -m bool }
```

```suggestion
    http-request lua.auth-intercept be_auth_request /api/authz/forward-auth HEAD * authorization,proxy-authorization,remote_user,remote-user,remote-groups,remote-name,remote-email - if protected-frontends
    http-request redirect location %[var(txn.auth_response_location)] if protected-frontends !{ var(txn.auth_response_successful) -m bool }
```

Indenting looks a little off here.